src/git/doGitSetRefOnCommit.ts

```diff
@@ -15,7 +16,9 @@ export default async function (
   }
   await spawn(
     "git",
-    ["fetch", remote, refPair].concat(force ? ["--force"] : []),
+    ["fetch", remote, refPair].concat(
+      getOptionalBooleanFlagArgument("--force", force)
```

Comment:
the earlier code `force ? ["--force"] : []` seems more readable than the new function

Reply:
Yeah, I hesitated to refactor this bit.
I don't mind changing it if indeed this add complexity

Reply:
👍 Yes I think it would be simplify